ada_2_fhir-r4/fhir/2_fhir_fhir_include.xsl

```diff
@@ -557,4 +558,42 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
         <xsl:value-of select="substring($logicalId, $startingLoc, $lengthLogicalId)"/>
     </xsl:function>
     
+    <xd:doc>
+        <xd:desc>Returns true (boolean) if the date or dateTime is in the future. Defaults to false. Input must be a value that is castable to a date or dateTime.</xd:desc>
```

Comment:
Input may also be empty, right?

Reply:
Mmm, you are right. It would then fall into the otherwise block... and result into _false_. 
Will adjust the docs.